src/controllers/Product.controller.ts

```diff
@@ -81,3 +84,62 @@ export const getAvailableProducts = async (req: Request, res: Response) => {
     }
   }
 };
+
+// Seller getting all items
+export const getAllSellerItems = async (req: Request, res: Response) => {
+  try {
+    const allItems = await getAllItemsService();
+    return res.status(200).json({
+      statusCode: 200,
```

Comment:
```suggestion
      status: 200,
```

Reply:
These response messages thought not fundamentally different, they are structurally different and could lead to confusion down the road because we have all been using ```status``` instead of ```statusCode```.
If you change it right now, it will be confusing.
Check other instances in the code-base to see the format we are using.

Reply:
I have done w.r.t your suggestions check again. 
and thank you for your review comment.